api/models.py

```diff
@@ -191,30 +213,22 @@ class Node(DatabaseModel):
         description='Dictionary with names mapping to node associated \
 URLs (e.g. URL to binaries or logs)'
     )
-    created: Optional[datetime] = Field(
+    created: datetime = Field(
         default_factory=datetime.utcnow,
         description='Timestamp of node creation'
     )
-    updated: Optional[datetime] = Field(
+    updated: datetime = Field(
         default_factory=datetime.utcnow,
         description='Timestamp when node was last updated'
     )
-    timeout: Optional[datetime] = Field(
+    timeout: datetime = Field(
+        default_factory=DefaultTimeout(hours=6).get_timeout,
```

Comment:
```suggestion
        default_factory=DefaultTimeout().get_timeout,
```

Reply:
I really don't like this, the default timeout value should be explicitly stated in the field definition.  Otherwise, changing the default value in the DefaultTimeout class will implicitly change the defaults in the models which would be very hard to track.  Also it means it's hard to figure out the default value when looking at a model definition as you need to look up the DefaultTimeout class implementation etc.

Reply:
Makes sense.